units/patch_intent_test.go

```diff
@@ -27,6 +28,10 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+const (
+	VersionCollection = "versions"
```

Comment:
It looks like the units collection already imports model so I'm not sure why we can't just use model.VersionCollection